admin/src/api/post/content-types/post/schema.json

```diff
@@ -74,9 +74,15 @@
       "type": "text",
       "required": true
     },
+    "new_content": {
```

Comment:
What is the use of `new_content` and `new_blog_content`? Is both required?

Reply:
Yes, **new_content** works instead of **content** and **new_blog_content** works instead of **blog_content**.
Actually I forgot we used content field.